### DISCUSSION TITLE
『切换到dev分支之后』这句怎么添加空格最合适

### DISCUSSION THREAD

**Discussion by Delayless** [sparanoid/chinese-copywriting-guidelines#143] [Q&A] [answered]
想问下，『切换到dev分支之后』这句话怎样添加空格比较好
1. 切换到 dev分支 之后
2. 切换到 dev 分支之后
3. 切换到 dev分支之后

诸如此类的更绕的还有『切换到远程master分支之后』

**Comment by sgalal** [comment, marked as answer]
<div lang="zh-CN">

中英文之间添加空格即可。

- 切换到 dev 分支之后
- 切换到远程 master 分支之后

</div>
